src/lib/reif.pl

```diff
@@ -23,16 +23,7 @@
 U. Neumerkel and S. Kral. https://arxiv.org/abs/1607.01590 [cs.PL]. July 2016.
 */
 
-
-:- meta_predicate
```

Comment:
Can you keep this directive from the original? It would be ideal to keep the differences as small as possible.

Reply:
It looks like `meta_predicate` doesn't accept a list in Scryer, is it Ok to just repeat it multiple times? Like so:
```prolog
:- meta_predicate(if_(1, 0, 0)).
:- meta_predicate(cond_t(1, 0, ?)).
:- meta_predicate(tfilter(2, ?, ?)).
:- meta_predicate(tpartition(2, ?, ?, ?)).
:- meta_predicate(','(1, 1, ?)).
:- meta_predicate(;(1, 1, ?)).
:- meta_predicate(tmember(2, ?)).
:- meta_predicate(tmember_t(2, ?, ?)).
```

Reply:
Please consider filing this problem as an issue, it should have been resolved as part of https://github.com/mthom/scryer-prolog/issues/1586 which may have been closed prematurely. Thank you a lot!

Reply:
I tried to do it myself, but because my Rust knowledge is non-existent, I just created a separate task for someone who actually knows how to do it :)

Reply:
Yes perfect, thank you a lot! For the time being, we can use individual directives so that we continue with this work. It is good to have it as an issue.